### DISCUSSION TITLE
Why are "forms" not supported?

### DISCUSSION THREAD

**Discussion by alexander-lopez-s** [lab-brussels-1/loruki-group-4#25] [General]
I got an error after running the command _run npm validate:html_
It looks like the `<input>` tag is not supported. How can I override this rule?

![Screenshot 2022-03-30 at 22 11 48](https://user-images.githubusercontent.com/97364283/160924383-4bde4389-b8de-42dc-8453-fcbc0b7ef949.png)

**Comment by TheRick88** [comment]
A classmate had the same error today, I had it last week, we both were able to fix it after reading this link:

https://stackoverflow.com/questions/469059/button-vs-input-type-button-which-to-use

Hope it helps.

**Reply by alexander-lopez-s** [reply]
The article clarifies the difference between <input> and <button> but does not say how to override the rule 😞

**Reply by TheRick88** [reply]
yeah it means that you need to replace `input type=button` with <button></button> at least thats what helped me and also Maria today with the same issue.

**Comment by bermarte** [comment]
add `"prefer-button": [0]` to `.htmlvalidate.json`
```
{
  "extends": ["html-validate:recommended"],
  "rules": {
    "void-style": [0],
    "no-trailing-whitespace": [0],
    "prefer-button": [0]
  }
}
```
But it's better to use the `<button>` element.
